controllers/comments-controller.js

```diff
@@ -0,0 +1,18 @@
+const { selectCommentsByArticleID } = require("../models/comments-model");
+const { selectArticlesByID } = require("../models/articles-model");
+
+exports.getCommentsByArticleID = (request, response, next) => {
+  const { article_id } = request.params;
+  const promises = [selectCommentsByArticleID(article_id)];
+
+  if (article_id) {
```

Comment:
The beauty of promise.all is if you put selectArticlesByID first and that rejects, the whole thing rejects. So no need for this if condition, if your query is malformed you'd have error handling on that first promise to protect this.